.github/workflows/build.yaml

```diff
@@ -19,13 +23,15 @@ jobs:
   repos:
     uses: hubverse-org/hub-dashboard-control-room/.github/workflows/get-installations.yaml@main
     with:
-      newbies: ${{ toJSON(github.event.client_payload.newbies) }}
+      newbies: '${{ toJSON(github.event.client_payload.newbies) }}'
+      repo: '${{ inputs.repo }}'
     secrets:
       id: ${{ vars.APP_ID }}
       key: ${{ secrets.PRIVATE_KEY }}
   site:
     needs: [repos]
     strategy:
+      fail-fast: false
```

Comment:
`fail-fast: false` means that all matrix jobs are allowed to run until they complete. By default, GitHub sets this to `true`, which means that if one matrix job fails, any of the others are automatically cancelled.